.circleci/config.yml

```diff
@@ -146,7 +146,6 @@ workflows:
           name: dev-promote-patch
           orb-name: netgaintechnology/deploy-orb
           ssh-fingerprints: d0:62:d6:03:0d:e0:43:4a:a2:4e:b5:59:d4:9f:00:a0
-          cleanup-tags: true
```

Comment:
these are needed to clean up the tags that trigger the integration workflow

Reply:
Found it. Below is the PR where this was merged into prod by CircleCI.
https://github.com/CircleCI-Public/orb-tools-orb/pull/64
Notes from RP:

> cleanup-tags isn't unused. it's a boolean for whether or not to do so. the cleanup-tags-pattern param then allows you to specify which tags to cleanup (if cleanup-tags is true). however, it is unused in the dev-promote-prod job in particular, where tags are always cleaned up & the user specifies only which tags should be removed. thank you !